File: src/Solution104_MaxDepthOfBT.java
import utils.TreeNode;

import java.util.Stack;

public class Solution104_MaxDepthOfBT {

    public int maxDepth(TreeNode root) {
        if (root == null)
            return 0;
        return 1 + Math.max(maxDepth(root.left), maxDepth(root.right));
    }
}

File: src/Solution104_MaxDepthOfBT.kt
import utils.TreeNode

import java.util.Stack

class Solution104_MaxDepthOfBT {

    fun maxDepth(root: TreeNode?): Int {
        return if (root == null) 0 else 1 + Math.max(maxDepth(root.left), maxDepth(root.right))
    }

    companion object {

        @JvmStatic
        fun main(args: Array<String>) {

        }
    }
}
